content/en/docs/releases/feature-stages/index.md

```diff
@@ -25,17 +25,16 @@ labels mean.
+| Support               | No guarantees on backward compatibility. The feature may be removed at any time without notice. There is no commitment from the Istio community to improve, maintain or complete the feature and the feature may be dropped entirely in a newer release.                                                                                                                                                                                                                                                                                                                                                                           | No guarantees on backward compatibility. There is no commitment from Istio to complete the feature. The feature may be dropped entirely in a later software release at any time without notice.                                                                                                                                                                                                                                                                                                                                                    | The overall feature will not be dropped, though details may change. Istio commits to complete the feature, in some form, in a subsequent Stable version. Typically this will happen within 3 months, but sometimes longer. Releases should simultaneously support two consecutive versions (e.g. v1alpha1 and v1beta1; or v1beta1 and v1) for at least one supported release cycle (typically 3 months) so that users have enough time to upgrade and migrate resources.                                                                                                                                                                                                                                                                                                                                                                                                                        | The feature will continue to be present for many subsequent releases.                                                                  |
```

Comment:
Tweak the first column a little.
```suggestion
| Support               | No guarantees on backward compatibility. There is no commitment from the Istio community to improve, maintain or complete the feature and the feature may be dropped entirely in a later software release at any time without notice.
| No guarantees on backward compatibility. There is no commitment from Istio to complete the feature. The feature may be dropped entirely in a later software release at any time without notice.                                                                                                                                                                                                                                                                                                                                                    | The overall feature will not be dropped, though details may change. Istio commits to complete the feature, in some form, in a subsequent Stable version. Typically this will happen within 3 months, but sometimes longer. Releases should simultaneously support two consecutive versions (e.g. v1alpha1 and v1beta1; or v1beta1 and v1) for at least one supported release cycle (typically 3 months) so that users have enough time to upgrade and migrate resources.                                                                                                                                                                                                                                                                                                                                                                                                                        | The feature will continue to be present for many subsequent releases.                                                                  |
```